package.json

```
@@ -2,7 +2,7 @@
     "name": "vscode-azureresourcegroups",
     "displayName": "Azure Resources",
     "description": "%azureResourceGroups.description%",
-    "version": "0.8.1-alpha.0",
+    "version": "0.8.1-alpha.2",
```

```suggestion
    "version": "0.8.1-alpha.0",
```

We'll bump the version once we're ready to release